Tasks/duplicate.js

```diff
@@ -1,14 +1,11 @@
 // Return an array without duplicates
+'use strict';
 
-duplicate = (value, N) => {
-  if (N <= 0) return [];
-  else {
-    res = [];
-    for (let i = 0; i < N; i++) {
-      res[i] = value;
-    }
-    return res;
-  }
+const duplicate = (value, amount) => {
+    const array = [];
+    for (let i = 0; i < amount; i++)
+      array[i] = value;
+    return array;
```

Comment:
I'd recommend to send indentation of two spaces (makes it a little bit easier to read).